kustomize/startMinikubeDeployLocalDev.sh

```diff
@@ -18,6 +18,37 @@ if [ $status -ne 0 ]; then
   kustomize build overlays/minikube | kubectl apply -f -
 fi
 
+rhoas login
+if [ "$(rhoas status -o json | jq --raw-output 'select( .kafka.name == "minikube-test-instance" )')" == "" ]; then
```

Comment:
If I'm not wrong, the name instance must be unique across all the users in managed kafka: it means that only one person can create `minikube-test-instance` and use it -> I think this will work only for the person that creates it